docs_src/microservices/device/details/DeviceProfiles.md

```diff
@@ -4,4 +4,126 @@ title: Device Service - Device Profiles
+| defaultValue | String         | N         | Default value to when no value present for set command. If present, should be compatible with the Type field                                                                                                                                                                                                                                         |
+| mask         | Uint64         | N         | A binary mask which will be applied to an integer reading. Only valid where Type is one of the unsigned integer types                                                                                                                                                                                                                                |
+| shift        | Int64          | N         | A number of bits by which an integer reading will be shifted right.Only valid where Type is one of the unsigned integer types                                                                                                                                                                                                                        |
+| scale        | Float64        | N         | A factor by which to multiply a reading before it is returned.Only valid where Type is one of the integer or float types                                                                                                                                                                                                                             |
+| offset       | Float64        | N         | A value to be added to a reading before it is returned. Only valid where Type is one of the integer or float types                                                                                                                                                                                                                                   |
+| base         | Float64        | N         | A value to be raised to the power of the raw reading before it is returned. Only valid where Type is one of the integer or float types                                                                                                                                                                                                               |
+| assertion    | String         | N         | String string value to which a reading (after processing) is compared. If the reading is not the same as the assertion value, the device's operating state will be set to disable. This can be useful for health checks.                                                                                                                             |
```

Comment:
```suggestion
| assertion    | String         | N         | String string value to which a reading (after processing) is compared. If the reading is not the same as the assertion value, the device's operating state will be set to disabled. This can be useful for health checks.                                                                                                                             |
```